lib/network/modules/LayoutEngine.js

```diff
@@ -196,34 +196,48 @@ class LayoutEngine {
       if (positionDefined < 0.5 * this.body.nodeIndices.length) {
         let MAX_LEVELS = 10;
         let level = 0;
-        let clusterThreshold = 100;
+        let clusterThreshold = 150;
+        //Performance enhancement, cluster edges need only be simpel straight lines
+        let clusterOptions = {
+          clusterEdgeProperties:{
+            smooth: {
```

Comment:
Just a quick note, this option is only used during the clustering phase of the initial layout. This is not user visible and doesn't influence the end result, it only halves the size of the nodes list during  clustering...